### ISSUE TITLE
 PLEASE READ (github migration form Mozilla AMO issues)

### ISSUE THREAD

**Issue by KevinMind** [mozilla/addons-frontend#12971]
## TLDR;
During the pending migration (2nd May 2024) we recommend you to [pause notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications) on these repositories:
mozilla/addons -> https://github.com/mozilla/addons
mozilla/addons-frontend -> https://github.com/mozilla/addons-frontend
mozilla/addons-server -> https://github.com/mozilla/addons-server
## What
We are planning to migrate our github issues to a single team repo on 2nd May 2024 at 17:00 CET
## Why
We are doing this in order to reduce overhead managing our backlog and better track our work as a team.
## What it means for you
If you are receiving a notification about this email, it means you are subscribed to notifications on one of these repositories. During the migration, many issues will be updated and this may cause email notifications to be sent to you.
We are working on a way to limit the number of notifications that are sent, but in the event that this proves impossible to prevent, we wanted to communicate with you before so you can adjust your settings.
We will also add  a label to issues that are migrated "migration-2024" which may help you identify issues updated due to the migration or due to updates.
You can also manage your email notification settings [here](https://docs.github.com/en/enterprise-cloud@latest/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#filtering-email-notifications)
Thank you for being a member of our community, please come say hi in our matrix channel: https://matrix.to/#/#amo:mozilla.org

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDFRNT-337
